### DISCUSSION TITLE
Am I being naive thinking that this might work?

### DISCUSSION THREAD

**Discussion by domsinclair** [leokhoa/laragon#625] [Q&A]
I'm trying to creat a laravel + jetstream app using quick app.

I tried the following, and various combinations thereof

```
LaravelJetstreamWithInertia =composer create-project laravel/laravel %s --prefer-dist && composer require laravel/jetstream && php artisan jetstream:install inertia --ssr --dark
```

No joy unfortunately.  Is this sort of thing possible?

**Comment by Pen-y-Fan** [comment]
You could move the jetstream and inertia parts to a batch file:

```bat
composer require laravel/jetstream
php artisan jetstream:install inertia --ssr --dark
```

Create the batch file, say **jetstream.bat** and place it in the laragon/usr directory. Use the Quick app to create laravel, once created the terminal is in the new directory, then call the batch file > **jetstream.bat**. This would be a two-step process, but still a little more automated.

**Reply by domsinclair** [reply]
Thanks for the suggestion.  It works in so much as the first line will run but it doesn't proceed from there.  It's not the end of the world if it has to be done manually but I'm naturally lazy at heart and love to automate things like this.
